js/data/languages.js

```diff
@@ -1187,3 +1200,13 @@ export function getTranslationLangByCode(code) {
 
   return getIndexedTranslationLang()[code];
 }
+
+export function getTrumboLangFileNameByCode(code) {
```

Comment:
Instead of having to loop through that list everytime, this could take advantage of the `getIndexedTranslationLang` function which will index that list by code and cache it. The list doesn't change, so caching it will ensure it only has to be looped through once instead of on every call.

You could basically change:

```
const lang = translationLangs.find((x) => x.code === code);
return lang.trumbowyg || null;
```

to

```
return getIndexedTranslationLang()[trumbowyg];
```

Reply:
Just updated that example code... there was a mistake in there.

Reply:
@rmisio This makes sense - but I can't reuse the getIndexedTranslationLang as is since it doesn't include the trumbowyg property. Would you prefer that I modify getIndexedTranslation so that the trumbowyg property is included _or_ make a separate cache?

Reply:
sure does. I overlooked that re-mapping before. Thanks!